src/AffectedUserFlow/CodeInput/CodeInputScreen.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FunctionComponent } from "react"
+import { View, StyleSheet } from "react-native"
+import { SafeAreaView } from "react-native-safe-area-context"
+
+import { usePermissionsContext, ENEnablement } from "../../PermissionsContext"
+import CodeInputForm from "./CodeInputForm"
+import EnableExposureNotifications from "./EnableExposureNotifications"
+
+import { Colors } from "../../styles"
+
+const CodeInputScreen: FunctionComponent = () => {
+  const { exposureNotifications } = usePermissionsContext()
+
+  const hasExposureNotificationsEnabled = (): boolean => {
+    const enabledState: ENEnablement = "ENABLED"
+    return exposureNotifications.status[1] === enabledState
```

Comment:
This feels like it needs to know too much about the exposure notification object to ask a question to a provider we are already creating. Would it make sense(probably not on this PR) to add a way to ask this directly in that context?

Reply:
That's a nice idea. So we would move this logic to the context and just get something like `userHasExposureNotificationsEnabled` directly from the context?

Reply:
Maybe, it could also make sense to parse it into an object, that might hold all the info, like having props for when it is available and not enabled or not available at all. It depends on the use case. I would shy away from having the method be too specific but maybe, in this case, specific is exactly what we want.